tests/acceptance/pageObjects/trashbinPage.js

```diff
@@ -34,6 +34,20 @@ module.exports = {
         .click('@dialogConfirmBtnEnabled')
         .waitForAjaxCallsToStartAndFinish()
         .waitForElementNotPresent('@dialog')
+    },
+    isActionButtonVisible: async function (batchButton) {
+      let result = false
+      batchButton = batchButton.toLowerCase()
+      let actionButtonSelector = null
+      if (batchButton === 'restore') {
+        actionButtonSelector = '@restoreSelectedButton'
+      } else if (batchButton === 'delete') {
+        actionButtonSelector = '@deleteSelectedPermanentlyButton'
+      }
+      await this.isVisible(actionButtonSelector, (res) => {
+        result = res.value
+      })
+      return result
```

Comment:
nice implementation here, returning the visibility rather than just verifying the visibility is true or false :+1: